src/microbe_stage/PlacedOrganelle.cs

```diff
@@ -379,6 +388,42 @@ public void ResetGrowth()
         }
     }
 
+    public void ReParentShapes(Microbe to, Vector3 offset)
+    {
+        if (to == currentShapesParent)
+            return;
+
+        var hexes = Definition.GetRotatedHexes(Orientation).ToArray();
+
+        for (int i = 0; i < shapes.Count; i++)
+        {
+            Vector3 shapePosition = Hex.AxialToCartesian(hexes[i]) + Hex.AxialToCartesian(Position);
+
+            if (ParentMicrobe.Colony != null)
+                shapePosition = shapePosition.Rotated(Vector3.Up, ParentMicrobe.Rotation.y);
```

Comment:
So the rotation comes here? This is confusing to me, I'd prefer one million percent if the rotation was also passed into this method, instead of this going to look at it behind the back of the other code.